frontend/src/app/commonComponents/Header.scss

```diff
@@ -17,4 +21,117 @@
   .usa-navbar {
     margin-right: 1rem;
   }
+
+  .role-tag {
+    // $theme-color-prime-gray:
+    color: #71767a;
+    margin-top: 0.5rem;
+  }
+
+  .header-link-icon {
+    // $theme-color-primary
```

Comment:
I'm kind of a CSS newbie. Why not use the USWDS color token here?

Reply:
Good question! We disabled the color tokens in component Sass files because adding them in required some additional overhead that we didn't want to deal with. Might want to revisit this as using the color tokens becomes more widespread or suggest folks refactor their Sass so that anything using the tokens are in the main `PrimeStyles.scss` file

cc @nickclyde